packages/cli/doc-plugin-preview/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "@modern-js/doc-plugin-preview",
+  "version": "2.19.1",
+  "description": "The Documentation Framework of Modern.js.",
+  "homepage": "https://modernjs.dev/doc-tools/",
+  "bugs": "https://github.com/web-infra-dev/modern.js/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/web-infra-dev/modern.js",
+    "directory": "packages/cli/doc-plugin-preview"
+  },
+  "license": "MIT",
+  "jsnext:source": "./src/index.ts",
+  "types": "./dist/index.d.ts",
+  "main": "./dist/index.js",
+  "scripts": {
+    "dev": "modern-lib dev",
+    "build": "modern-lib build",
+    "reset": "rimraf ./**/node_modules",
+    "lint": "modern-lib lint",
+    "change": "modern-lib change",
+    "bump": "modern-lib bump",
+    "pre": "modern-lib pre",
+    "change-status": "modern-lib change-status",
+    "gen-release-note": "modern-lib gen-release-note",
+    "release": "modern-lib release",
+    "new": "modern-lib new",
+    "test": "vitest run --passWithNoTests",
+    "upgrade": "modern-lib upgrade"
+  },
+  "engines": {
+    "node": ">=14.17.6"
+  },
+  "eslintIgnore": [
+    "node_modules/",
+    "dist/"
+  ],
+  "dependencies": {
+    "@mdx-js/mdx": "2.2.1",
+    "@modern-js/doc-core": "workspace:*",
+    "@modern-js/utils": "workspace:*",
+    "qrcode.react": "^3.1.0"
+  },
+  "devDependencies": {
+    "@arco-design/web-react": "^2.46.0",
```

Comment:
@arco-design/web-react should be a dependency

Reply:
i have bundled it